git_tarballs

```diff
@@ -256,6 +256,14 @@ def update_changes_file(package, changes):
         f.close()
 
 
+def _check_filenames(*filenames):
+    for filename in filenames:
+        basename = os.path.basename(filename)
+        if os.path.abspath(filename) != os.path.abspath(basename):
+            # no arbitrary filename, please
+            sys.exit("%s: illegal filename" % filename)
```

Comment:
can you make that a little bit more explicit ? "illegal filename" is not very helpful if thrown. maybe say something like "--filename needs to be pointing to a file in the current directory" ?